lib/node_modules/@stdlib/math/base/special/nanmin/README.md

```diff
@@ -0,0 +1,118 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# nanmin
+
+> If one operand is NaN, the other operand is always returned; otherwise, perform normal minimum
```

Comment:
Maybe instead:

```suggestion
> Return the minimum value, ignoring NaN.
```

The description should start by explaining what the function does. Also good to aim at being succinct. Notice also that we require a period at the end of the description.